package motor_goal_pos_list.msgs;

import "motor_goal_position.proto";

message MotorGoalPositionList {
    repeated motor_goal_position.msgs.MotorGoalPosition motor_goal_pos_list = 1;
}